wrangler-core/src/main/java/co/cask/directives/parser/ParseSimpleDate.java

```diff
@@ -47,7 +49,7 @@
 public class ParseSimpleDate implements Directive {
   public static final String NAME = "parse-as-simple-date";
   private String column;
-  private SimpleDateFormat format;
+  private SimpleDateFormat formatter;
```

Comment:
why not use the DateTimeFormatter you used in the other directive?

Reply:
This is because `FormatDate` formats `ZonedDateTime` to `String`.